### DISCUSSION TITLE
This user account is blocked

### DISCUSSION THREAD

**Discussion by lyquangthai1993** [vercel/community#4035] [Account Issue] [answered]
### Question

Please help me unlock when I try to sign in with github account, thanks you.
My account email is lyquangthai1993@gmail.com
Best regard.

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
